lib/datasource/pod/index.ts

```diff
@@ -32,13 +40,26 @@ function shardParts(lookupName: string): string[] {
 
 function releasesGithubUrl(
   lookupName: string,
-  opts: { account: string; repo: string; useShard: boolean }
+  opts: {
+    hostURL: string;
```

Comment:
```suggestion
    hostURL?: string;
```

From the codepath below, it looks like it's optional?